### ISSUE TITLE
Doc perms migration fails in InnoDB Cluster deployments

### ISSUE THREAD

**Issue by p-kimberley** [gchq/stroom#4513]
In branch 7.5, the migration `07.05.00.005 - Orphaned Doc Perms` fails with error: `The table does not comply with the requirements by an external plugin.`

The error root cause is due to the table `doc_permission_backup_V07_05_00_005` being created by the migration script with an ID column that's not a primary key. This is easily fixed by amending the table creation code in the migration.

This issue only affects MySQL InnoDB Cluster deployments, which require a primary key column in each table in order for group replication to work.

**Comment by at055612** [comment]
Before you run it again you will need to run

```sql
delete from cross_module_schema_history where version = '07.05.00.005';
```
to remove the migration from the log so it will run again.

Also ensure table `doc_permission_backup_V07_05_00_005` doesn't exist in the db.

**Comment by at055612** [comment]
Fixed in >7.5-beta.19
